src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -617,7 +655,10 @@ export function getNextEvents(currentStatus) {
           },
         ]
       };
-    case STATUS_TYPES.pendingForm9:
+    }
+    case STATUS_TYPES.pendingForm9: {
+      const certificationDuration = makeDurationHeader(details.certificationTimeliness);
+      const ssocDuration = makeDurationHeader(details.ssocTimeliness);
```

Comment:
Are these going to be camelCase like this, or snake_case? Might be worth putting a TODO here to make sure once we get integrated with the api.